packages/discovery-react-components/src/components/SearchFacets/components/FacetsGroups/CollapsibleFacetsGroup.tsx

```diff
@@ -70,8 +83,34 @@ export const CollapsibleFacetsGroup: FC<CollapsibleFacetsGroupProps> = ({
     setIsCollapsed(!isCollapsed);
   };
 
+  let facetsByCategory: FieldFacetsByCategory = {};
+  if (hasCategories) {
+    facetsByCategory[`${facetsLabel}`] = { categories: {} };
+    if (isSelectableQueryTermAggregationResult(facets)) {
+      facets.map(result => {
+        const resultType = result!.aggregations![0].results![0].key;
+        if (resultType in facetsByCategory[`${facetsLabel}`].categories) {
```

Comment:
i feel like this if-else can be DRY-ed up a bit, but we can leave that for a later PR